improver_tests/wxcode/wxcode/test_utilities.py

```diff
@@ -511,5 +507,40 @@ def test_raises_exception(self):
             interrogate_decision_tree("kittens")
 
 
+@pytest.mark.parametrize(
```

Comment:
Somehow by adding this test I think you've removed the tests you had for get_parameter_names. I think?

Reply:
Thanks for pointing this out. I think I probably made a mistake when trying to correct the earlier mistake of committing the change to the wrong branch. I have (hopefully) put these tests back without causing any more problems.

Reply:
Yep they have reappeared :)